EfEnumToLookupTests/Model/CandyBag.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.ComponentModel.DataAnnotations.Schema;

```

Comment:
good choice of test names :+1: :smile:

Reply:
It was between a candy bag or perhaps an engine with parts, but candy won :)